demo/yun/components/examples/FormatDate.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
```

Comment:
Whether we should add a prefix namespace to i?

Reply:
I think it's possible. Initially, I named it DateTest, and I have been struggling with how to do it better, so it's entirely fine to go with your idea.

Reply:
`FormatDate` is same with funtion in valaxy.
So maybe we can call it `TestFormatDate` or others not `FormatDate`.